geode-docs/managing/monitor_tune/performance_controls_controlling_socket_use.html.md.erb

```diff
@@ -25,17 +25,22 @@ The conserve-sockets setting indicates whether application threads share sockets
 
 You configure conserve-sockets for the member as a whole in `gemfire.properties`. Additionally, you can change the sockets conservation policy for the individual thread through the API.
 
-When conserve-sockets is set to false, each application thread uses a dedicated thread to send to each of its peers and a dedicated thread to receive from each peer. Disabling socket conservation requires more system resources, but can potentially improve performance by removing socket contention between threads and optimizing distributed ACK operations. For distributed regions, the put operation, and destroy and invalidate for regions and entries, can all be optimized with conserve-sockets set to false. For partitioned regions, setting conserve-sockets to false can improve general throughput.
+By default, conserve-sockets is set to false: each application thread uses a dedicated thread to send to each of its peers and a dedicated thread to receive from each peer.
+This default setting optimizes system performance, but does incur some overhead in terms of system resources.
+
+Leaving socket conservation in its default disabled state alleviates socket contention between threads and optimizes distributed ACK operations. For distributed regions, the put operation, and destroy and invalidate for regions and entries, all perform optimally with conserve-sockets set to false. For partitioned regions, setting conserve-sockets to false enhances general throughput.
+
```

Comment:
I had to read that second sentence (starting with For distributed regions) a few times before I understood it.

Reply:
Rewritten.